packages/wdio-cli/src/utils.ts

```diff
@@ -222,7 +222,13 @@ export async function renderConfigurationFile (answers: ParsedAnswers) {
     const tplPath = path.join(__dirname, 'templates/wdio.conf.tpl.ejs')
     const filename = `wdio.conf.${answers.isUsingTypeScript ? 'ts' : 'js'}`
     const renderedTpl = await renderFile(tplPath, { answers })
-    return fs.promises.writeFile(path.join(process.cwd(), filename), renderedTpl)
+    return fs.promises.writeFile(
+        path.join(
+            process.cwd(),
+            answers.isUsingTypeScript ? 'test' : '', filename
+        ),
+        renderedTpl
+    )
```

Comment:
I think we want to revert this change because we don't want the wdio.conf being generated within `/test/wdio.conf.ts`.

Reply:
Hi @christian that was the discussion in previous comment to keep both config and tsconfig in test folder

Reply:
This must be a misunderstanding then. I remember we decided to have the `tsconfig.json` within the test folder but keep `wdio.conf.ts` in the root directory. With this change the current wdio.conf having this config:

```ts
        tsNodeOpts: {
            transpileOnly: true,
            project: 'test/tsconfig.json' // <-- wdio.conf needs to be in the root directory.
        }
```

wouldn't work. Therefore let's keep the wdio.conf in the root directory and the tsconfig.json in /test

Reply:
Hi @chritian , the IDE will throw error if tsconfig and wdio config is not in the same directory . That's why we decided keep both in test folder for typescript

Reply:
@christian-bromann  What do you think ? :(